llvm/test/MC/AArch64/SME2p2/utmopa-diagnostics.s

```diff
@@ -0,0 +1,77 @@
+// RUN: not llvm-mc -triple=aarch64 -show-encoding -mattr=+sme2p2 2>&1 < %s| FileCheck %s
+
+// --------------------------------------------------------------------------//
+// Invalid ZA register
+
+sutmopa  za4.s, {z30.b-z31.b}, z31.b, z31[3]
```

Comment:
I think this should have contained the tests for `utmopa`, we have these in `sutmopa-diagnostics.s`.